.goreleaser.yaml

```diff
@@ -0,0 +1,38 @@
+project_name: csproto
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - id: protoc-gen-fastmarshal
+    main: ./cmd/protoc-gen-fastmarshal
+    binary: protoc-gen-fastmarshal
+    flags:
+      - -trimpath
+    ldflags:
+      - '-X main.version={{ .Version }} -X main.commit={{ .ShortCommit }} -X main.date={{ .Date }} -X main.builtBy=goreleaser'
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      # - windows
```

Comment:
Any particular reason not to build for Windows?

Reply:
yeah if not using CGo, windows should be trivial. Also I believe we should build ARM for all pertinent platforms (Linux and Darwin)

Reply:
Nothing specific.  I did Mac and Linux because I was able to verify the resulting binaries were valid locally.

Reply:
Updated to do the full matrix of Linux/Mac/Windows X 386/amd64/arm/arm64 (w/ arm 6 & 7)